examples/models/transform_jitter.py

```diff
@@ -15,7 +15,7 @@
 normal = Jitter(width=0.2, distribution="normal")
 uniform = Jitter(width=0.2, distribution="uniform")
 
-p = figure(x_range=(0, 4), y_range=(0,10))
+p = figure(x_range=(0, 4), y_range=(0,10),toolbar_location='right')
```

Comment:
This toolbar_location shouldn't be necessary, but I'm fine to merge with this in.